java/src/main/java/com/splunk/support/lambda/ServerTimingHeader.java

```diff
@@ -20,7 +20,7 @@
 import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.propagation.TextMapSetter;
-import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
```

Comment:
Hmmm, this seems like a problem, but we. use this in our java distro as well. Will have to think about it.